src/metabase/api/search.clj

```diff
@@ -410,9 +422,18 @@
             (if (number? v)
               (not (zero? v))
               v))]
-    (let [search-query      (full-search-query search-ctx)
+    (let [search-sql-tokens (scoring/sql-tokenize (scoring/normalize (:search-string search-ctx)))
+          search-query      (full-search-query search-ctx)
           _                 (log/tracef "Searching with query:\n%s" (u/pprint-to-str search-query))
-          reducible-results (db/reducible-query search-query :max-rows search-config/*db-max-results*)
+          search-sql        (hsql/format search-query
+                                         :quoting             (db/quoting-style)
+                                         :allow-dashed-names? (not (db/automatically-convert-dashes-and-underscores?))
+                                         :params              {:search-string (:search-string search-ctx)
+                                                               :search-tokens (str "'" (str/join "|" search-sql-tokens) "'")})
+          reducible-results (jdbc/reducible-query (db/connection)
+                                                  search-sql
+                                                  (merge {:max-rows search-config/*db-max-results*}
+                                                         @(var-get #'db/default-jdbc-options)))
```

Comment:
Aren't the arguments of `merge` swapped?

Reply:
Correct. Nice one. Fixed https://github.com/metabase/metabase/pull/24171/commits/01080e552d238f083bba818f846f1e40690e620d